src/web/app/src/components/SearchInput/AuthorInput.tsx

```diff
@@ -1,82 +1,134 @@
-      borderRadius: `4rem`,
+      borderRadius: ({ isListOpen }) => (isListOpen ? '2rem 2rem 0 0' : '4rem'),
+      transition: theme.transitions.create(['border-radius'], {
+        duration: '.15s',
+      }),
       borderColor: theme.palette.info.main,
-      borderWidth: `2px`,
+      borderWidth: '2px',
+      padding: '1rem !important',
+      '&:hover fieldset': {
+        borderColor: `${theme.palette.info.main} !important`,
+      },
+      '&.Mui-focused fieldset': {
+        borderColor: `${theme.palette.info.main} !important`,
+      },
+      '& fieldset': {
+        borderColor: theme.palette.info.main,
+        borderWidth: '2px',
+        borderBottom: ({ isListOpen }) =>
+          isListOpen ? `none` : `2px solid ${theme.palette.info.main}`,
+        boxShadow: ({ isListOpen }) =>
+          isListOpen ? `${theme.palette.info.main} 0px 1px 4px` : 'none',
+      },
+    },
+    label: {
+      color: `${theme.palette.text.secondary} !important`,
+      fontSize: '1.4rem',
+    },
+    popper: {
+      boxSizing: 'border-box',
+      borderWidth: '1px 2px 2px',
+      borderColor: ({ isListOpen }) => (isListOpen ? theme.palette.info.main : 'transparent'),
+      borderStyle: 'solid',
+
+      overflow: 'hidden',
+      borderRadius: '0 0 2rem 2rem',
+      boxShadow: ({ isListOpen }) =>
+        isListOpen ? `${theme.palette.info.main} 0px 1px 4px` : 'none',
+      '& > div': {
+        backgroundColor: theme.palette.background.default,
+        borderRadius: 0,
+      },
     },
     listbox: {
-      fontSize: '1.2rem',
       color: theme.palette.text.primary,
-      backgroundColor: theme.palette.background.paper,
-      '& :hover': {
-        color: theme.palette.text.secondary,
-        backgroundColor: theme.palette.background.default,
-        border: '2px solid',
+      borderRadius: 0,
+      fontFamily: 'inherit',
+      fontSize: '1.4rem',
+    },
+    clear: {
+      color: theme.palette.text.primary,
+      '& svg': {
+        fontSize: '2rem',
+      },
+    },
+    option: {
+      borderRadius: '3px',
+      '&.Mui-focused': {
+        backgroundColor: `${theme.palette.background.paper} !important`,
       },
     },
   })
 );
 
-const fetchResults = async (url: string | null) => {
-  if (url) {
-    try {
+interface AuthorInputInterface {
+  text: string;
+  setText: Dispatch<SetStateAction<string>>;
+  labelFor: string;
+}
+
+const AuthorInput = ({ text, setText, labelFor }: AuthorInputInterface) => {
+  const [isListOpen, setIsListOpen] = useState(false);
+  const [author, setAuthor] = useState(text);
+
+  const shouldFetch = () => author.length > 0;
+  const prepareUrl = () => `${searchServiceUrl}/authors/autocomplete/?author=${text}`;
+  const { data: options } = useSWR<AutocompleteOption[]>(
+    shouldFetch() ? prepareUrl() : null,
+    async (url) => {
       const res = await fetch(url);
       if (res.ok) {
         const results = await res.json();
         // return up to first 10 results
```

Comment:
If we're going to only consume max 10 results, we might as well limit this on the server-side.  We can do this in a follow-up, or you can change the back-end in this PR.

Reply:
Yeah, I dropped the border radius and will leave the backend change out for later.